docs/Deploy-Using-Docker.md

```diff
@@ -0,0 +1,169 @@
+
+To access the `mysql` shell on a docker container simply execute the following command:
+
+##### Template
+
+```bash
+docker exec -it "{CONTAINER_NAME}" mysql -p"{MYSQL_ROOT_PASSWORD}"
+```
+
+Where:    
+- **{CONTAINER_NAME}**: The name of your container instance _i.e **cbio-DB**_.
+- **{MYSQL_ROOT_PASSWORD}**: The root password for the MySQL installation. For password restrictions please read carefully this [link](http://dev.mysql.com/doc/refman/5.7/en/user-names.html).
+
+#### 2.2.4 Useful Resources
+[MySQL Docker Hub](https://hub.docker.com/_/mysql/)    
+[MySQL Docker Github](https://github.com/docker-library/docs/tree/master/mysql)
+
+## 3. cBioPortal Setup
+
+### 3.1 Run the cBioPortal docker container 
+
+In a docker terminal type the following command:
+
+##### Template
+
+```bash
+docker run -d --name ${CONTAINER_NAME} \
+    --restart=always \
+    --net=${DOCKER_NETWORK_NAME} \
+    -p ${PREFERRED_EXTERNAL_PORT}:8080 \
+    -v ${PATH_TO_portal.properties}:/cbioportal/src/main/resources/portal.properties:ro \
+    -v ${PATH_TO_log4j.properties}:/cbioportal/src/main/resources/log4j.properties:ro \
+    -v ${PATH_TO_settings.xml}:/root/.m2/settings.xml:ro \
+    -v ${PATH_TO_context.xml}:/usr/local/tomcat/conf/context.xml:ro \
+    -v ${PATH_TO_CBIOPORTAL_LOGS}:/cbioportal_logs/ \
+    -v ${PATH_TO_TOMCAT_LOGS}:/usr/local/tomcat/logs/ \
+    -v ${PATH_TO_STUDIES}:/cbioportal_studies/:ro \
+    cbioportal/cbioportal:${TAG}
+```
+
+Where:    
+- **${CONTAINER_NAME}**: The name of your container instance, _i.e **cbioportal**_.
+- **${DOCKER_NETWORK_NAME}**: The name of your network, _i.e **cbio-net**_.
+- **${PREFERRED_EXTERNAL_PORT}**: The port that the container internal port will be mapped to, _i.e **8306**_.
+- **${PATH_TO_portal.properties}**: The external path were portal.properties are stored.
+- **${PATH_TO_log4j.properties}**: The external path were log4j.properties are stored.
+- **${PATH_TO_settings.xml}**: The external path were settings.xml is stored.
+- **${PATH_TO_context.xml}**: The external path were context.xml is stored.
+- **${PATH_TO_CUSTOMIZATION}**: The external path were customization files are stored.
+- **${PATH_TO_CBIOPORTAL_LOGS}**: The external path where you want cBioPortal Logs to be stored.
+- **${PATH_TO_TOMCAT_LOGS}**: The external path where you want Tomcat Logs to be stored.
+- **${PATH_TO_STUDIES}**: The external path where cBioPortal studies are stored.
+- **${TAG}**: The cBioPortal Version that you would like to run, _i.e **latest**_.
+
+### 3.2 Run DB Migrations
+
+Update the seeded database schema to match the cBioPortal version in the image.
+
+:warning: Running the following docker command will most likely make your database irreversibly incompatible with older versions of the portal code.
+
+##### Template
+
+```bash
+docker exec -it ${CONTAINER_NAME} \
+migrate_db.py -p /cbioportal/src/main/resources/portal.properties -s /cbioportal/db-scripts/src/main/resources/migration.sql
```

Comment:
I prefer to run scripts in their own, clean containers separate from the Catalina process like we do in the version we use at The Hyve, but this will do for now.